colcon_bundle/installer/apt.py

```diff
@@ -154,6 +154,10 @@ def setup(self):  # noqa: D102
                                'you set your keys correctly?')
         self._cache.open()
 
+        # Workaround for pip-requirements not installing python-pip
+        self.add_to_install_list('python-pip')
```

Comment:
This package doesn't exist on Ubuntu Focal anymore.

Reply:
thanks, I'll update the commit

Reply:
Removed